api/services/protocols/local.js

```diff
@@ -25,7 +25,7 @@ const crypto    = require('crypto');
  */
 exports.register = async function (req, res, next) {
   const email    = req.param('email'),
-    name = req.param('name'),
+    name = req.param('username'),
```

Comment:
Was this an intentional change? If so, we should change the AuthController tests as well, because they're currently failing since they pass the wrong parameter.